<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora de Indutância e Capacitância</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbInductor">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>16</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Indutor</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbInductorFrequency">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>48</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Frequência</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbInductorInput">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>96</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">lbInductorInput</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbInductorResult">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>96</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">lbInductorResult</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edInductorFrequencyVal">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>64</y>
      <width>112</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="edInductorInputVal">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>112</y>
      <width>112</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edInductorResultVal">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>112</y>
      <width>112</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cbInductorFrequencyUnit">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>64</y>
      <width>56</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cbInductorInputUnit">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>112</y>
      <width>56</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbInductorResultUnit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>112</y>
      <width>56</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btInductorAlternate">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>111</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Alternar</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/icon_swap.png</normaloff>images/icon_swap.png</iconset>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="btInductorCopy">
    <property name="geometry">
     <rect>
      <x>424</x>
      <y>111</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Copiar</string>
    </property>
    <property name="text">
     <string>Copiar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/icon_copy.png</normaloff>images/icon_copy.png</iconset>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCapacitorFrequencyUnit">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>216</y>
      <width>56</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbCapacitor">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>168</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Capacitor</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btCapacitorAlternate">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>263</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Alternar</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/icon_swap.png</normaloff>images/icon_swap.png</iconset>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbCapacitorInput">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>248</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">lbCapacitorInput</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edCapacitorInputVal">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>264</y>
      <width>112</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCapacitorInputUnit">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>264</y>
      <width>56</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edCapacitorFrequencyVal">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>216</y>
      <width>112</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCapacitorResultUnit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>264</y>
      <width>56</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btCapacitorCopy">
    <property name="geometry">
     <rect>
      <x>424</x>
      <y>263</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Copiar</string>
    </property>
    <property name="text">
     <string>Copiar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/icon_copy.png</normaloff>images/icon_copy.png</iconset>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbCapacitorFrequency">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>200</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Frequência</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCapacitorResult">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>248</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">lbCapacitorResult</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edCapacitorResultVal">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>264</y>
      <width>112</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="btAbout">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>8</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/icon_info.png</normaloff>images/icon_info.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lbInductorImage">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>10</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/inductor.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbCapacitorImage">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>162</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/capacitor.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="btHelp">
    <property name="geometry">
     <rect>
      <x>424</x>
      <y>8</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/icon_question.png</normaloff>images/icon_question.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edInductorFrequencyVal</tabstop>
  <tabstop>cbInductorFrequencyUnit</tabstop>
  <tabstop>edInductorInputVal</tabstop>
  <tabstop>cbInductorInputUnit</tabstop>
  <tabstop>btInductorAlternate</tabstop>
  <tabstop>edInductorResultVal</tabstop>
  <tabstop>cbInductorResultUnit</tabstop>
  <tabstop>btInductorCopy</tabstop>
  <tabstop>edCapacitorFrequencyVal</tabstop>
  <tabstop>cbCapacitorFrequencyUnit</tabstop>
  <tabstop>edCapacitorInputVal</tabstop>
  <tabstop>cbCapacitorInputUnit</tabstop>
  <tabstop>btCapacitorAlternate</tabstop>
  <tabstop>edCapacitorResultVal</tabstop>
  <tabstop>cbCapacitorResultUnit</tabstop>
  <tabstop>btCapacitorCopy</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>8</number>
  </property>
  <property name="gridDeltaY">
   <number>8</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
